packages/editor-ui/src/components/WorkflowTagsContainer.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+	<TagsContainer
+		:tag-ids="tagIds"
+		:tags-by-id="tagsById"
+		:limit="limit"
+		:clickable="clickable"
+		:responsive="responsive"
+		:hoverable="hoverable"
+		@click="onClick"
+	/>
+</template>
+
+<script lang="ts" setup>
+import { computed, defineProps, defineEmits } from 'vue';
```

Comment:
No need to import `defineProps` and `defineEmits`, in fact it leads to a warning:
![CleanShot 2024-08-27 at 14 02 31](https://github.com/user-attachments/assets/cbb332b0-1224-4bb7-86d2-07772e40c156)